Vagrantfile

```diff
@@ -129,6 +134,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     app.vm.provision "ansible" do |ansible|
+      ansible.extra_vars = extra_vars
```

Comment:
`lookup` will pull from the environment of the machine running Ansible. It should work just like the Ruby `ENV` inside the Vagrant definition, but inside Ansible land.